plugins/cohere/credentials.go

```diff
@@ -0,0 +1,111 @@
+package cohere
+
+import (
+	"context"
+
+	"encoding/json"
+	"fmt"
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func Credentials() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.Credentials,
+		DocsURL:       sdk.URL("https://docs.cohere.com/reference/config"),
+		ManagementURL: sdk.URL("https://dashboard.cohere.ai/"),
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.Authtoken,
+				MarkdownDescription: "JWT used to authenticate to Cohere stored in disk.",
+				Secret:              true,
+			},
+			{
+				Name:                fieldname.Email,
+				MarkdownDescription: "Email used to authenticate to Cohere.",
+				Optional:            true,
+			},
+			{
+				Name:                fieldname.URL,
+				MarkdownDescription: "URL of the operator server",
+				Optional:            true,
+			},
```

Comment:
I see these are marked as optional, could you point me to any documentation as to what the defaults are / what happens when these are not provisioned?

Reply:
- I have updated Email as optional
https://docs.cohere.com/reference/config
- Install co cli
- curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/cohere-ai/co/main/install.sh | sh
- login through `co auth login --email=email@covbase.com`
- then enter password 
`Now check ~/.command/config`

Reply:
Indeed the email and url are optional while only the JWT is mandatory